make/docker-compose.clair.yml

```diff
@@ -17,7 +17,7 @@ services:
         aliases:
           - postgres
     container_name: clair-db
-    image: postgres:9.6.3
+    image: vmware/harbor-postgresql:9.6.3
```

Comment:
Is it postgresql 9.6.3?
Also, shall we call it vmware/postgresql:$version-photon to follow the convention?